packages/ibm-products/src/components/Datagrid/Datagrid/DatagridHeaderRow.js

```diff
@@ -143,12 +143,13 @@ const HeaderRow = (datagridState, headRef, headerGroup) => {
                   [`${blockClass}__resizableColumn`]: header.getResizerProps,
                   [`${blockClass}__isResizing`]: header.isResizing,
                   [`${blockClass}__sortableColumn`]:
-                    datagridState.isTableSortable,
+                    datagridState.isTableSortable && header.id !== 'spacer',
                   [`${blockClass}__isSorted`]: header.isSorted,
                 },
                 header.getHeaderProps().className
               )}
               key={header.id}
+              aria-hidden={header.id === 'spacer' && 'true'}
```

Comment:
I don't understand how `&& 'true'` is supposed to work.

Reply:
Basically if it's a spacer column, aria-hidden will be true, otherwise it will be false for all other columns.

Reply:
Otherwise the spacer column is included as a column header, for example without this change
```js
screen.getAllByRole('role');
```
this gives back all of the columns including the spacer column.

Reply:
_Right!_ It will assign the string "true" if hidden.id is "spacer".

Mondays.